cmd/lotus-storage-miner/market.go

```diff
@@ -858,18 +858,25 @@ var dealsPendingPublish = &cli.Command{
 			return xerrors.Errorf("getting pending deals: %w", err)
 		}
 
-		w := tabwriter.NewWriter(os.Stdout, 2, 4, 2, ' ', 0)
-		_, _ = fmt.Fprintf(w, "ProposalCID\tClient\tSize\n")
+		if len(pending.Deals) > 0 {
+			w := tabwriter.NewWriter(os.Stdout, 2, 4, 2, ' ', 0)
+			_, _ = fmt.Fprintf(w, "Publish period:             %s\n", pending.PublishPeriod)
```

Comment:
Maybe also add `(ends in %s)` with the time from now to the period end

Reply:
👍  added

Reply:
```
$ ./lotus-miner storage-deals pending-publish
Publish period:             1h0m0s (ends in 55m11s)
First deal queued at:       2021-02-08 11:08:39.222417 +0100 CET
Deals will be published at: 2021-02-08 12:08:39.222417 +0100 CET
...
```